packages/nc-gui/layouts/base.vue

```diff
@@ -97,17 +97,17 @@ hooks.hook('page:finish', () => {
                   </nuxt-link>
                 </a-menu-item>
 
-                <a-menu-divider class="!m-0" />
-                <!--                <a-menu-item v-if="isUIAllowed('superAdminAppStore')" key="0" class="!rounded-t">
+                <!-- <a-menu-divider class="!m-0" />
+                <a-menu-item v-if="isUIAllowed('superAdminAppStore')" key="0" class="!rounded-t">
                   <nuxt-link
                     v-e="['c:settings:appstore', { page: true }]"
                     class="nc-base-menu-item group !no-underline"
                     to="/admin/users"
                   >
                     <MdiShieldAccountOutline class="mt-1 group-hover:text-accent" />&nbsp;
 
-                    &lt;!&ndash; todo: i18n &ndash;&gt;
-                    <span class="prose group-hover:text-primary">Account management</span>
+                    &lt;!&ndash; {{`${$t('general.todo')}: i18n`}} &ndash;&gt;
```

Comment:
this is a TODO tag. we don't need to translate this text .. You can simply remove this line as you've translated L110.